### DISCUSSION TITLE
C'est quoi une Directive ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#23] [General]
# C'est quoi une Directive ? | Bonjour Angular

C'est quoi une Directive dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/directive/

**Comment by AurelienPREVOST** [comment]
cela pourrait permettre de modifier les couleurs de l'habillage d'une SPA en fonction de la personne qui se connecte et de ses choix dans une interface de personnalisation? Si l'on stocke l'info en DB
